ArduCopter/Log.cpp

```diff
@@ -547,8 +582,23 @@ const struct LogStructure Copter::log_structure[] = {
 // @Field: aY: Target acceleration, Y-Axis
 // @Field: aZ: Target acceleration, Z-Axis
 
-    { LOG_GUIDEDTARGET_MSG, sizeof(log_GuidedTarget),
-      "GUID",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+    { LOG_GUIDED_POSITION_TARGET_MSG, sizeof(log_Guided_Position_Target),
+      "GPOS",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+
+// @LoggerMessage: GATT
+// @Description: Guided mode attitude target information
+// @Field: TimeUS: Time since system startup
+// @Field: Type: Type of guided mode
+// @Field: Roll: Target attitude, Roll
+// @Field: Pitch: Target attitude, Pitch
+// @Field: Yaw: Target attitude, Yaw
+// @Field: RollRate: Roll rate in centidegrees/s
+// @Field: PitchRate: Pitch rate in centidegrees/s
+// @Field: YawRate: TYaw rate in centidegrees/s
+// @Field: Thrust: Thrust -1 to 1 
+
+    { LOG_GUIDED_ATTITUDE_TARGET_MSG, sizeof(log_Guided_Attitude_Target),
+      "GATT",  "QBfffffff",    "TimeUS,Type,Roll,Pitch,Yaw,RollRate,PitchRate,YawRate,Thrust", "s-hhhkkk-", "F-BBBBBB-" , true },
```

Comment:
Roll is defined -180deg to 180deg and pitch -90 to 90deg (though not exactly 90deg or bad things happen ;) )
The yaw is sent in as `guided_angle_state.yaw_cd = wrap_180_cd(ToDeg(guided_angle_state.yaw_cd) * 100.0f);` so we should use degrees `d` for that too

```suggestion
      "GATT",  "QBfffffff",    "TimeUS,Type,Roll,Pitch,Yaw,RollRate,PitchRate,YawRate,Thrust", "s-dddkkk-", "F-BBBBBB-" , true },
```